tests/unit/sdk/test_client.py

```diff
@@ -83,6 +83,16 @@ async def test_method_count(clients, mock_query_repository_count, client_type):
     assert count == 5
 
 
+@pytest.mark.parametrize("client_type", client_types)
```

Comment:
Not sure if this test is really checking anything since the payload is static
ideally we should add an integration tests for it